README.md

```diff
@@ -67,6 +72,12 @@ nrpe::plugin { 'check_mem':
 }
 ```
 
+Full documentation for the `nrpe::plugin` type is available in the [reference](REFERENCE.md#nrpeplugin) documentation.
+
+## Reference
+
+This module has been documented with [puppet-strings](https://github.com/puppetlabs/puppet-strings).  See [REFERENCE.md](REFERENCE.md).
```

Comment:
I'd first link to the reference and then tell you how it's generated.

Reply:
Fixed.  I've also added the command to generate it to the `Development` section.